hack/azure-credentials.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -ex
```

Comment:
i'm usually hesitant to add a `-x` in a bash script that the caller doesn't have control, but i ran this script from your branch and the output was actually nicer (more informative).